### ISSUE TITLE
[Bug] go get is not working

### ISSUE THREAD

**Issue by r-franzke** [ccremer/paperless-cli#52]
## Describe the bug

$ go install github.com/ccremer/paperless-cli@latest
```
go: github.com/ccremer/paperless-cli@latest: version constraints conflict:
        github.com/ccremer/paperless-cli@v0.5.0: parsing go.mod:
        module declares its path as: github.com/ccremer/clustercode
                but was required as: github.com/ccremer/paperless-cli
```
## Logs

If applicable, add logs to help explain your problem.
```console

```

## To Reproduce

1. ...

## Environment (please complete the following information):

- App Version: e.g. v1.0

**Comment by ccremer** [comment]
oops, sorry about this one, this is an error that originally roots back to a copy paste mistake that I didn't notice.

I've fixed this now in v0.5.1. Thanks for reporting! Please try again and reopen if still not working.

**Comment by r-franzke** [comment]
Thank you!
